### PR TITLE
NIFI-1867 Improve ModifyBytes to make it easy to remove all FlowFile content (0.x)

### PR DESCRIPTION
**Pull request by jskora**
* Add "Remove All Content" property, related functionality, and tests to ModifyBytes processor to allow deletion of all flow file content.
* Removed @Ignore annotation on class and unnecessary EOL translation of test data.  Because ModifyBytes treat input as binary data, not text, line endings don't matter as long as they byte offsets are calculated correctly.

(For 0.x branch, a separate PR for 1.x branch will follow.)

### CODE REVIEW

File: nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/ModifyBytes.java [hunk 1]
```diff
@@ -74,6 +74,13 @@
             .addValidator(StandardValidators.DATA_SIZE_VALIDATOR)
             .defaultValue("0 B")
             .build();
+    public static final PropertyDescriptor REMOVE_ALL = new PropertyDescriptor.Builder()
+            .name("Remove All Content")
+            .description("Remove all content from the FlowFile superseding Start Offset and End Offset properties.")
+            .required(true)
+            .addValidator(StandardValidators.BOOLEAN_VALIDATOR)
+            .defaultValue("false")
```
Comment (pvillard31) [inline nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/ModifyBytes.java]: Could you add
````
.allowableValues("true", "false")
````
in such a case?

Comment (jskora) [inline nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/ModifyBytes.java]: Will do.

**Comment by jskora** [comment]
@pvillard31, I committed the .allowableValues change to this and the 1.x pull https://github.com/apache/nifi/pull/890.

**Comment by pvillard31** [comment]
+1, checked improvement, full build w/ contrib-check, LGTM.
Merged in 0.x.
@jskora, could you manually close this PR? Thanks!

**Comment by jskora** [comment]
Closed by 0.x branch commit f89bc9efd8b8458b5bfd6a1b4045ce8230117ff4.

Thanks @pvillard31

**Pull request closed without merging**